### ISSUE TITLE
I want to organize my data in a well-structured hierarchy of folders and give access to specific datasets to specific client

### ISSUE THREAD

**Issue by sriram-dsl** [clearml/clearml#1165]
i have very sensitive data 
so, i dont want give acces of those datasets  to some persons,
but they should access other folders

something like in ceph

**Comment by ainoam** [comment]
@sriram-dsl ClearML does not manage storage access privilege (Well, its enterprise offering offers additional modules to support such capabilities).
To address this use case you can configure your storage architecture and privilege policy (e.g. using ceph) and then use clearml-data to log your datasets as references only (see [`--links`](https://clear.ml/docs/latest/docs/clearml_data/clearml_data_cli#add)) so these privileges are maintained.

Does this make sense?

**Comment by sriram-dsl** [comment]
Even we are also thinking to do same
If possible can you share, any related documentation

**Comment by ainoam** [comment]
@sriram-dsl This is the relevant documentation: [--links](https://clear.ml/docs/latest/docs/clearml_data/clearml_data_cli#add) parameter to `clearml-data add`.
What are you missing?

**Comment by ainoam** [comment]
@sriram-dsl The point I'm trying to make, is that using `--links` will make it so the data itself is not stored in ClearML but rather on your storage of choice, and so regardless of having access to the ClearML dataset your storage access policy will dictate who can actually access the data (i.e. I can see the dataset, but I can't access the files unless I have privilege to those files on the storage server).
Role based access control is available as part of the ClearML enterprise offering.

**Comment by sriram-dsl** [comment]
I agree with you. I have used it for S3, and it worked. Our team is looking
to launch a Ceph cluster and push the data to Ceph to limit access.
Currently, we are using NAS for our storage. I want to know how to launch
Ceph on NAS and mount the Ceph-deployed NAS folder on
`opt/clearml/data/fileserver`.

**Comment by sriram-dsl** [comment]
If I receive extended help, like relevant documents, it is appreciated. If
not, no issues either. We are currently occupied with other matters, but we
will address it soon.
